opensrp-chw-core/src/main/java/org/smartregister/chw/core/utils/ReportUtils.java

```diff
@@ -77,7 +80,12 @@ else if (monthlyTally != null) {
     }
 
     public static void addEvent(Map<String, String> values, String formSubmissionId, String formName, String tableName) throws JSONException {
-        JSONObject form = FormUtils.getFormUtils().getFormJson(formName);
+        JSONObject form = null;
+        try {
+            form = FormUtils.getFormUtils().getFormJson(formName);
+        } catch (IOException e) {
+            Timber.e(e);
+        }
         JSONObject stepOne = form.getJSONObject(org.smartregister.chw.anc.util.JsonFormUtils.STEP1);
```

Comment:
This would result on an NPE, we should execute all the code referencing the form within the try catch